src/http/server.rs

```diff
@@ -66,30 +73,103 @@ fn create_allowed_headers(headers: &HeaderMap, allowed: &BTreeSet<String>) -> He
 
   new_headers
 }
+
+fn load_cert(filename: &str) -> Result<Vec<rustls::Certificate>, std::io::Error> {
```

Comment:
Can we use the async implementation of tokio to read file asynchronously

Reply:
Sure; I need to keep the `BufReader` from the `std::io` though, because `rustls_pemfile` needs a `std::io::BufRead` to read from